Insung-Jo/level_0/암호 해독.js

```diff
@@ -0,0 +1,11 @@
+function solution(cipher, code) {
+    let array = cipher.split("");
+    let answer = []
+    
+    for(let i = 0; i <= array.length; i++){
+        if(i % code === 0 && i != 0) {
```

Comment:
```suggestion
      i !== 0 
```
여기는 실수로 느슨하게 한거 같네요.